### ISSUE TITLE
Include Azure IoT Gateway SDK dlls

### ISSUE THREAD

**Issue by terrymandin** [Azure/iot-gateway-opc-ua#6]
Hi,

The Azure IoT Gateway SDK build is not working for me.  See: https://github.com/Azure/azure-iot-gateway-sdk/issues/87

I think it would be easier for us to get this working in the field if the libraries you use with your latest checkin were included in github with your project.  And even better, if there were a setup program we could run to install the OPC UA gateway rather than building the gateway ourselves.

**Comment by barnstee** [comment]
Please post this issue in the GW SDK repro. Checking in binaries in GitHub is a bad idea in general. I'm waiting for NuGet packages for the GW SDK, then this pain will go away.

**Comment by terrymandin** [comment]
OK, I agree checking in binaries is a bad idea, its just frustrating not to be able to get going.  Perhaps we could put an OPC UA gateway install (MSI) on one of the Microsoft websites? I think it would help in its adoption.   My customer is reluctant to adopt this open source technology due to issues like this.  They are looking for a supported "product".

I already created an issue with the Gateway SDK.

**Comment by barnstee** [comment]
OK, cool. While the GW SDK is a supported product, the OPC UA client module is only a reference implementation and provided "as is" as the NuGet packages we use for the OPC UA stack are based on the OPC Foundation's reference stack (see NiGet package lic terms!). However, there are plenty of commercial gateway solutions connecting to Azure either being currently built or available already from the usual suspects, e.g. Softing, Unified Automation, Matrikon, etc. Please reach out to them for a supported product.

**Comment by barnstee** [comment]
Forgot to mention Kepware and Iconics.

**Comment by terrymandin** [comment]
OK, will do, thanks!

**Comment by terrymandin** [comment]
Sorry, one more question.  I don't want to create an issue as it is probably something I am doing wrong.

I was able to get the required dlls and lib files from the gateway sdk.  I moved them to the appropriate lib or binding directory.  The program is not running as it appears that it cannot load the "dotnet" module in line 331 of module_loader.c.  I checked and that file is in the binding directory, and is also being copied to the bin\debug directory.  Any thoughts on what I am doing wrong?  I am not much of a C developer unfortunately.

Perhaps I should be using the "classic" OPC connector instead?  I am trying to make a demo to pull OPC UA data and push it to the IoT Hub for our upcoming "Polaris" sales event.
